modules/core/src/main/java/org/apache/ignite/internal/processors/query/property/QueryBinaryProperty.java

```diff
@@ -53,18 +53,15 @@
     /** Result class. */
     private Class<?> type;
 
-    /** */
-    private volatile int isKeyProp;
+    /** Defines where value should be extracted from : cache entry's key or value. */
```

Comment:
Unused imports `org.apache.ignite.internal.util.typedef.internal.U` and `org.jetbrains.annotations.Nullable`

Reply:
Fixed